installer/config.yml

```diff
@@ -85,3 +85,170 @@ playout:
   # The sample size for recordings.
   # > default is 16
   record_sample_size: 16
+
+stream:
+  # Inputs sources.
+  inputs:
+    # Main harbor input.
+    main:
+      # Harbor input public url. If not defined, the value will be generated from
+      # the [general.public_url] hostname and the input port and mount.
+      url:
```

Comment:
Unsure about the naming of this fields, maybe url is too generic. Maybe we should use public_url or stream_url ?

Reply:
`public_url` I think is a good name here